model/client.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const uuidv4 = require('uuid-v4');
+const Client = module.exports = (socket) => {
```

Comment:
Changed from fat arrow to normal function declaration because when using 'this.'  with fat arrow it is just taking everything from the scope. So we do a normal function declaration because it will pass whatever is filled into the arguments.